vnext/Shared/OInstance.cpp

```diff
@@ -412,6 +406,7 @@ InstanceImpl::InstanceImpl(
           [[fallthrough]];
 #endif
         }
+        case JSIEngineOverride::Chakra:
         case JSIEngineOverride::ChakraCore:
```

Comment:
assert if ChakraCore set?

Reply:
We could potentially break Office automation scenarios that may have ChakraCore still set as their JS engine.

I think we send broad heads-up communications before taking any non-transparent measures.

Reply:
Would be nice to remove ChakraCore from Office first before making change here.

Reply:
At this point, ChakraCore is used in Office separately from RNW.
This dependency (RNW -> ChakraCore) is no longer used at all.